style.css

```diff
@@ -261,22 +265,149 @@ body.lock {
     font-size: 20px;
     right: 25px;
 }
-.arrow_left {
+.arrow-left {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(135deg);
 }
-.arrow_right {
+.arrow-right {
     border: solid var(--arrow-color);
     border-width: 0 2px 2px 0;
     display: inline-block;
     padding: 4px;
     transform: rotate(-45deg);
 }
 .content {
-    padding: 70px 0 0;
+    margin: 60px 0 0;
+}
+.carousel {
+    position: relative;
+    width: 100vw;
+    height: 150vw;
+    overflow: hidden;
+}
+.carousel__text {
+    position: absolute;
+    top: 50%;
+    left: 8%;
+    text-align: left;
+    display: active;
+    z-index: 20;
+}
+.carousel__text1 {
+    color: var(--element-color);
+    font-size: 9vw;
```

Comment:
Why did you decide to set font size based on viewport width? If you do this for text here then do it for all text so it properly scales up and down on the whole page.

Reply:
I did this so that text of the titles is better read on different devices. Of course I can change it in all places (but I will keep px in the desktop version).

Reply:
You can't have vw in once place and px another place, use same measurements as much as you can.

Reply:
Ok. I changed  px to vw in all fonts. And I will try to change as much as I can in other places later.

Reply:
How much is 9vw in pixels? I doubt that is the correct size for this font based on the provided design files.

Reply:
> How much is 9vw in pixels? I doubt that is the correct size for this font based on the provided design files.
about 35

Reply:
I think it was a bad experiment and I should change everything back to pixels